### DISCUSSION TITLE
Part Number for Right-Angle 4-pin Connectors?

### DISCUSSION THREAD

**Discussion by wmehilos** [tjko/fanpico#29] [Q&A] [answered]
Hi all, 

I looked all over the repo as best I could but couldn't find a part number for the right-angle 4-pin connectors for the fans, just the vertical ones. 

i know they're not terribly hard to find on digi-key/mouser and so on by making some best guesses but I rather like the look of the ones in the pictures

**Comment by tjko** [comment, marked as answer]
I couldn't find any from Digi-Key/Mouser either...  (maybe Molex does not even make one?)

Ones that I've been using have been ordered from Aliexpress, if you search for "KF2510" (or "KF2510 3+1P") you should  find those.
